File: openapi-test-java-server-quarkus-reactive-client-jdk/testapi/api/PetStatusChangedWebhook.java
package testapi.api;

import com.fasterxml.jackson.databind.JsonNode;
import io.smallrye.mutiny.Uni;

public interface PetStatusChangedWebhook {
  /** Called when a pet's status changes */
  Uni<Void> onPetStatusChanged(JsonNode body);
}

File: openapi-test-kotlin-server-quarkus-reactive-client-mp-reactive/testapi/api/PetStatusChangedWebhook.kt
package testapi.api

import com.fasterxml.jackson.databind.JsonNode
import io.smallrye.mutiny.Uni
import java.lang.Void

sealed interface PetStatusChangedWebhook {
  /** Called when a pet's status changes */
  fun onPetStatusChanged(body: JsonNode): Uni<Void>
}